src/pages/Index.js

```diff
@@ -1,8 +1,95 @@
+/* eslint-disable import/no-extraneous-dependencies */
 import React from 'react';
 import { Link } from 'react-router-dom';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import 'photoswipe/dist/photoswipe.css';
+import { Gallery, Item } from 'react-photoswipe-gallery';
 
 import Main from '../layouts/Main';
 
+const galleryImages = [
+  {
+    key: 1,
+    alt: 'catdetector',
+    src: '/images/projects/catdetector.jpg',
+    width: '1200',
+    height: '803',
+    thumbnail: '/images/projects/catdetector.jpg',
```

Comment:
If you have a smaller thumbnail you want to use, you could put its file path here. Note that I simply used the original image because the files you have in the `public/projects` folder are already small enough, I think.